### DISCUSSION TITLE
MRTK Quest

### DISCUSSION THREAD

**Discussion by SabinMGeorge** [microsoft/MixedRealityToolkit-Unity#9968] [Q&A]
Hi ,
How can I trigger system menu event from hand pinch gesture(MRTK quest)? There is no event mapping available in ArticulatedHandDefinition profile? 

Reference
![image](https://user-images.githubusercontent.com/62987118/121816821-124e5080-cc7e-11eb-8d74-5889328eb9e8.png)

**Comment by polar-kev** [comment]
Hi @SabinMGeorge, is your question how to bring up the Oculus system menu using a pinch gesture in MRTK? Or how to recognize a pinch gesture in MRTK similar to the one used to bring up the Oculus system menu? 

MRTK doesn't currently support custom gesture recognition, you'd have to manually implement the palm up pinch detection.
